groupAnagrams.js

```diff
@@ -0,0 +1,44 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/group-anagrams/
+
+const groupAnagrams = (strs) => {
+  let strsArray = strs;
+  const result = [];
+  for (let i = 0; i < strsArray.length; ) {
```

Comment:
Not sure why you've defined this `i` variable inside of this loop, it doesn't seem to change its value - hence you need to refactor it somehow (I'm not sure how exactly because I don't understand what it is).

Reply:
Corrected with new commit - [605fe2f](https://github.com/Femalopper/tasks/pull/6/commits/605fe2f4df6707932152c34ea64449794be424f5)